packages/app-desktop/gui/ConfigScreen/loadFonts.ts

```diff
@@ -0,0 +1,5 @@
+import fontList = require('font-list');
+
+fontList.getFonts({ disableQuoting: true })
+	.then((fonts: string[]) => process.send(fonts))
+	.catch((error: any) => console.error(error));
```

Comment:
Use async/await. How does it actually handle errors? It's printed to the console?

Reply:
ok no problem
Yes, it just prints the error to the console and the list just keeps showing `Loading...` like this
![error-state](https://github.com/laurent22/joplin/assets/113056556/e96b1a48-4ab0-468e-9d71-463fda9c3d21)